stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -136,7 +135,7 @@ extension ${Self}: LosslessStringConvertible {
   %if bits == 16:
     self.init(Substring(text))
   %else:
-    if #available(macOS 10.16, iOS 14.0, watchOS 7.0, tvOS 14.0, *) {
+    if #available(macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0, *) {
```

Comment:
@tbkka I think we made a change here that hasn't been pushed upstream for whatever reason--do you want to bring the two in sync now?

Reply:
Yes, we should sync these up.

Reply:
Do you want to put up a PR and tag me to review?